### DISCUSSION TITLE
Activeadmin v4 Chartkick integration

### DISCUSSION THREAD

**Discussion by imadilkhalil** [activeadmin/activeadmin#8261] [Help] [answered]
I am trying to add Chartkick to my app based off 4.0.0.beta4 branch but to no avail.

I have followed their [guide](https://chartkick.com/) for Importmap but the scripts are not picked up when I view the source in the browser. I assume that the `application.js` is not picked up by the asset pipeline.

Any help would be appreciated.

**Comment by wrozka** [comment]
@imadilkhalil I've used @barnaclebarnes solution from this discussion: https://github.com/activeadmin/activeadmin/discussions/8346#discussioncomment-10131266
to integrate Chartkick. Give it a try

**Comment by thisismydesign** [comment, marked as answer]
I solved it like this: https://stackoverflow.com/questions/78985376/activeadmin-v4-how-to-add-custom-js-via-importmaps/78985377#78985377
